web/src/main/java/org/springframework/security/web/servletapi/SecurityContextHolderAwareRequestFilter.java

```diff
@@ -42,13 +42,8 @@
  * A <code>Filter</code> which populates the <code>ServletRequest</code> with a request
  * wrapper which implements the servlet API security methods.
  * <p>
- * In pre servlet 3 environment the wrapper class used is
- * {@link SecurityContextHolderAwareRequestWrapper}. See its javadoc for the methods that
- * are implemented.
- * </p>
- * <p>
- * In a servlet 3 environment {@link SecurityContextHolderAwareRequestWrapper} is extended
- * to provide the following additional methods:
+ * {@link SecurityContextHolderAwareRequestWrapper} is extended to provide the following
```

Comment:
There is a separate ticket for this class, so let's go ahead and leave these changes out in this file out. Thank you, though!

Reply:
on it, wait a second, messed up my branch D: